runtime/doc/eval.txt

```diff
@@ -4703,6 +4703,7 @@ input({opts})
 		cancelreturn  ""       Same as {cancelreturn} from
 		                       |inputdialog()|. Also works with 
 		                       input().
+		highlight     nothing  Highlight handler: |Funcref|.
```

Comment:
How often is the handler called? Should mention this somewhere.

Typically we prefix VimL handlers with `on_` (Vim does `_cb` suffix). Maybe this should be called `on_input`.